ai/anthropic/v0/config/tasks.json

```diff
@@ -0,0 +1,298 @@
+{
+  "$defs": {
+    "multi_modal_content": {
```

Comment:
Important
Please change to kebab-case.
Same as other properties

Reply:
fixed